### DISCUSSION TITLE
Vessel History Separation of Records

### DISCUSSION THREAD

**Discussion by tamihuber** [paulwinterbauer/NBIC#395] [Q&A]
I thought I knew the separation criteria, but this one confused me. When does the separator show?
![image](https://user-images.githubusercontent.com/100441006/198125616-ad91a3ac-0b02-425d-b13e-9fe2b1598808.png)

**Comment by paulwinterbauer** [comment]
You mean the '18 days' right?  It's when the days between the records are a week or more.  But that includes NOA departure dates which can make it look wonky when they have noa arrivals from like '23-Sep -> 30-Oct' like this one.
